buzzmobile/tools/route_mapper/googlemapskey.py.example

```diff
@@ -1,2 +0,0 @@
-# Has to have Google Maps Static API enabled in API Console
-googlemapskey=''
```

Comment:
this file should not be deleted either.

Also can you revert the `world_model.png`? I can't make a comment on a BIN file, but that image is not what `world_model` looks like.